packages/core/src/events/handlers/index.ts

```diff
@@ -20,7 +24,7 @@ import { sendMagicLinkJob } from './sendMagicLinkHandler'
 
 type LatitudeEventGeneric<
   U extends keyof typeof EventHandlers,
-  T extends Record<string, unknown>,
+  T extends { userEmail?: string; workspaceId?: number; [key: string]: any },
 > = {
   type: U
   data: T
```

Comment:
Why adding `userEmail` to the generic type?
Is this just to humanize the events info?

Reply:
right at the beginning those weren't optional so that's why but you are totally right 👌🏼